runfinch/finch
# Fix: Env variable CONTAINERD_SNAPSHOTTER cleared on overlayfs

…actoring of override.yaml operations

Issue #, if available: Fix: Env variable CONTAINERD_SNAPSHOTTER cleared on overlayfs

*Description of changes:*

*Testing done:*
1. unit test
2. Add soci and check pull and push operation work with soci.
3. Add soci and check build works for soci
4. Add overlayfs and check it clears soci configs and sets everything to default.

Note: The e2e test is in progress locally.

- [x ] I've reviewed the guidance in CONTRIBUTING.md


#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
